Web/starter_project_iconics/components/Homepage/Landing.tsx

```diff
@@ -0,0 +1,38 @@
+import Image from 'next/image'
+import { BsArrowRightShort } from 'react-icons/bs'
+
+const Landing = () => {
```

Comment:
can you use syntac like React.FC to define the component? it is better to make it clear that we are looking at a component (there is a different one for pages too)